File: recap/Equality/Equality.java
import java.util.*;
public class Equality {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int a = scan.nextInt();
		scan.next();
		int b = scan.nextInt();
		scan.next();
		int c = scan.nextInt();
		if(a+b == c) {
			System.out.println("YES");
		}
		else {
			System.out.println("NO");
		}
	}
}

File: recap/Equality/Equality.cs
using System;
using System.Collections.Generic;
 
 
public class Solution {
 
    public static void Main(string[] args) {
        int a = nextInt();
        next();
        int b = nextInt();
        next();

        Console.WriteLine(a + b == nextInt() ? "YES" : "NO");
    }
 
 
    private static string[] __tokens;
    private static int __tidx;
    public static int nextInt() {
        return int.Parse(next());
    }
 
    public static string next() {
        if (__tokens == null || __tidx == __tokens.Length) {
            __tidx = 0;
            __tokens = Console.ReadLine().Split(' ');
        }
        return __tokens[__tidx++];
    }
}